include/zephyr/bluetooth/services/ots.h

```diff
@@ -250,6 +250,15 @@ enum {
 	BT_OTS_OACP_FEAT_ABORT      = 9,
 };
 
+/*
+ * @enum bt_ots_oacp_write_mode
+ * @brief Mode Parameter for OACP Write Op Code.
+ */
+enum bt_ots_oacp_write_mode {
+	BT_OTS_OACP_WRITE_OP_MODE_NONE = 0,
+	BT_OTS_OACP_WRITE_OP_MODE_TRUNCATE = BIT(1),
```

Comment:
Enums should probably use the same prefix for the enum name and the values, i.e. either add `op` to `bt_ots_oacp_write_mode`, or remove `OP` from `BT_OTS_OACP_WRITE_OP_MODE_`